### PR TITLE
Secure Admin Page and Press API

### PR DESCRIPTION
**Pull request by jm52262**

### CODE REVIEW

File: server/controllers/press.server.controller.js [hunk 1]
```diff
@@ -27,23 +28,29 @@ exports.createNew = function(req, res) {
...
+  }
 
 };
 
 exports.updateExisting = function(req, res) {
-    
-  var pressrelease = req.body;
 
-  var ObjectID = require('mongodb').ObjectID;
+  if(req.isAuthrnticated){
```
Comment (arsenalgunnershubert777) [inline server/controllers/press.server.controller.js]: mispelled Authenticated

**Review by arsenalgunnershubert777 — CHANGES_REQUESTED** [review]
one place mispelled isAuthenticated

File: server/controllers/press.server.controller.js (hunk 1 above)
Comment (jm52262) [reply server/controllers/press.server.controller.js]: Corrected

File: server/controllers/press.server.controller.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ exports.getAll = function(req, res) {
 
 exports.createNew = function(req, res) {
 
+  if(req.isAuthrnticated){
```
Comment (arsenalgunnershubert777) [inline server/controllers/press.server.controller.js]: typo here as well

**Review by arsenalgunnershubert777 — CHANGES_REQUESTED** [review]
one more typo

**Review by arsenalgunnershubert777 — APPROVED** [review]
looks good

**Pull request merged**